.pre-commit-config.yaml

```
@@ -0,0 +1,38 @@
+repos:
+-   repo: local
+    hooks:
+    -   id: isort
+        name: isort
+        entry: isort
+        require_serial: true
+        language: python
+        language_version: python3
+        types_or: [cython, pyi, python]
+        args: ['--filter-files']
+        minimum_pre_commit_version: '2.9.2'
+        additional_dependencies: [isort==5.13.2]
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+        exclude: ".*?(.hex|.clvm|.clib)"
```

https://github.com/Chia-Network/chia-blockchain/blob/bd3a638db10ee92a15f138ceaef06648c3c7b480/.pre-commit-config.yaml#L48

I don't recall the details around this...  but we might not want to get into them here.  Also, undo associated changes.

```suggestion
        exclude: ".*?(.hex|.clsp|.clvm|.clib)"
```